examples/sort/parallel_sort.py

```diff
@@ -0,0 +1,73 @@
+import ray
```

Comment:
Move the Ray import to last (and separated by a blank space from the other imports).